backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -788,19 +873,58 @@ export class UserResolver {
   }
 }
 
+async function findUserByEmail(email: string): Promise<DbUser> {
+  const dbUserContact = await DbUserContact.findOneOrFail(
+    { email: email },
+    { withDeleted: true, relations: ['user'] },
+  ).catch(() => {
+    logger.error(`UserContact with email=${email} does not exists`)
+    throw new Error('No user with this credentials')
+  })
+  const dbUser = dbUserContact.user
+  dbUser.emailContact = dbUserContact
+  return dbUser
```

Comment:
No it is not enough and returning the user and the userContact separatly will not solve the afterwards processing, because a user with an initialized userContact-attribute is needed.
The find with relations [user] returns a userContact with an initialized User, but in this user the userContact-attribute isn't initialized. That's why it is explicitly set before returning the user.